doc/source/categorical.rst

```diff
@@ -1126,12 +1138,12 @@ are not numeric data (even in the case that ``.categories`` is numeric).
 
 .. ipython:: python
 
-    s = pd.Series(pd.Categorical([1,2,3,4]))
+    s = pd.Series(pd.Categorical([1, 2, 3, 4]))
     try:
         np.sum(s)
-        #same with np.log(s),..
+        # same with np.log(s),..
     except TypeError as e:
-         print("TypeError: " + str(e))
+        print("TypeError: " + str(e))
```

Comment:
```suggestion
        print("TypeError:", str(e))
```

Reply:
thanks, changed this and applied the same to others.